File: cfgs/java_cfgs_foo/FIND_THE_NUMBER_OCCURRING_ODD_NUMBER_OF_TIMES.java
Nodes
1;special;;;
2;statement;int i;14;
3;statement;i = 0;15;
4;conditional;foo_1 < foo_2;16;
5;statement;int count = 0;18;
6;statement;int j = 0;19;
7;conditional;foo_1 < foo_2;20;
8;conditional;foo_1[foo_2] == foo_3[foo_4];22;
9;statement;count++;22;
10;statement;j++;21;
11;conditional;(foo_1 % 2) != 0;24;
12;statement;return foo_1[foo_2];24;
13;exit;;;
14;statement;i++;17;
15;statement;return -1;26;
Edges
0;2;
1;3;
2;4;
3;5;15;
4;6;
5;7;
6;8;11;
7;9;10;
8;10;
9;7;
10;12;14;
11;13;
12;
13;4;
14;13;

File: cfgs/pythoncfgs/FIND_THE_NUMBER_OCCURRING_ODD_NUMBER_OF_TIMES.py
Nodes
1;conditional;foo < foo;8;
2;statement;return -1;15;
3;statement;foo_1 = 0;9;
4;conditional;foo < foo;10;
5;conditional;foo_1 % 2 != 0;13;
6;statement;return foo_1[foo_2];14;
7;conditional;foo_1[foo_2] == foo_3[foo_4];11;
8;statement;foo_1 += 1;12;
9;exit;;
Edges
0;2;3;
1;9;
2;4;
3;5;7;
4;1;6;
5;9;
6;4;8;
7;4;
8;
